src/components/Alert/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ComponentType } from 'react';
+
+interface AlertProps {
+  Icon: ComponentType;
+  errorMessage: string[];
+}
+
+const Alert = ({ Icon, errorMessage }: AlertProps): JSX.Element => {
+  return (
+    <div className="alert">
```

Comment:
For proper styling, we should use one of the contextual classes [alert provides](https://getbootstrap.com/docs/5.0/components/alerts/#examples), like `alert-primary` 🎨

Reply:
I've changed it to AlertWarning ✌️

Reply:
It's in this commit https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/86f93d06bf0dcc9846e57c2a5a5389aa4bd26595